src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
-body {
+@import './utils/variables';
+@import './utils/mixins';
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include block-centering;
+}
+
+.stopwatch {
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+
+  @include block-centering;
+}
+
+.center {
+  z-index: 1;
+  height: $center-size;
+  width: $center-size;
+  background-color: $center-color;
+  border-radius: 50%;
+}
+
+.hand {
+  position: absolute;
+
+  &__minute,
+  &__second {
+    position: absolute;
+    transform-origin: bottom;
+  }
+
+  &__minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: $hand-minute-color;
+
+    top: -20vmin;
+    left: -1.5vmin;
+
+    animation: move $animation-minute-duration steps(60) infinite;
+  }
+
+  &__second {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: $hand-second-color;
+
+    top: -38vmin;
+    left: -0.75vmin;
+
+    animation: move $animation-second-duration linear infinite;
```

Comment:
Similar to the previous comment, ensure that `$animation-second-duration` includes a time unit or add one if necessary.